<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Название</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Жанр</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Год выпуска</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Издательство</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Автор</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>701</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>671</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addBookButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addAuthorButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить автора</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addVisitorButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить посетителя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GiveOutBookButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выдать книгу</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
